### PR TITLE
Move block

### PR DESCRIPTION
**Pull request by WanjikuMac**

### CODE REVIEW

File: lib/quadquizaminos_web/live/tetris_live.ex [hunk 1]
```diff
@@ -40,7 +40,9 @@ defmodule QuadquizaminosWeb.TetrisLive do
        categories: init_categories(),
        powers: [],
        adding_block: false,
-       coord_modal: false
+       coord_modal: false,
```
[the remark this answers is not in the document]
Comment (WanjikuMac) [reply lib/quadquizaminos_web/live/tetris_live.ex:43]: yes this is not in use

File: lib/quadquizaminos_web/live/tetris_live.ex [hunk 2]
```diff
@@ -40,7 +40,9 @@ defmodule QuadquizaminosWeb.TetrisLive do
...
+       block_coord: nil,
```
[the remark this answers is not in the document]
Comment (WanjikuMac) [reply lib/quadquizaminos_web/live/tetris_live.ex:44]: I agree with you.

File: lib/quadquizaminos_web/live/tetris_live.ex [hunk 3]
```diff
@@ -408,13 +414,41 @@ defmodule QuadquizaminosWeb.TetrisLive do
     {:noreply, socket}
   end
 
-  def handle_event("add_block", %{"x" => x, "y" => y}, socket) do
+  def handle_event("move_block", %{"x" => x, "y" => y, "color" => color}, %{assigns: %{moving_block: true}} = socket) do
+    x = String.to_integer(x)
```
Comment (sigu) [inline lib/quadquizaminos_web/live/tetris_live.ex:418]: Future refactor we can have this as a separate function, we have it repeated more than 3 times

**Review by sigu — APPROVED** [review]
Awesome 🎉

**Pull request merged**